controllers/controllers.js

```diff
@@ -36,6 +36,23 @@ exports.getArticleById = (req, res, next) => {
         res.status(200).send({article})
     })
     .catch((err) => {
+        if (err.code === '42703') {
+            err = {status: 404, msg: 'Invalid article_id'}
+        }
+        next(err)
+    })
+}
+
+exports.getAllCommentsFromArticleId = (req, res, next) => {
+    const { article_id } = req.params
+    selectAllCommentsFromArticleId(article_id)
+    .then (( comments ) => {
+        res.status(200).send({comments})
+    })
+    .catch((err) => {
+        if (err.code === '42703') {
```

Comment:
hmm I think I would maybe move this bit of code into your error handler, but it's up to you how you want to do it